src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -341,32 +341,32 @@ public bool DeleteMetadataForAttachment(string org, string appName, string id)
         }
 
         /// <summary>
-        /// Returns the service metadata for a service
+        /// Returns the serviceMetadata for an app.
```

Comment:
Either use ServiceMetadata or service metadata. Not serviceMetadata, this is a variable name.